string   filenamebase           # The first part of the filename, before the timestamp, e.g. 'test'.
string   filenamepart           # The filename (without the extension), with the base and timestamp, e.g. 'test20130309031415'.  This field is set by the code.
bool     arenastate
bool     video
bool     onlyWhileTriggered

